src/components/tables/Details/ShardInformation.tsx

```diff
@@ -1,127 +1,183 @@
-                                        />
+                                        >
+                                            <ExternalLink
+                                                link={intl.formatMessage({
+                                                    id: 'detailsPanel.shardDetails.docPath',
+                                                })}
+                                            >
+                                                <FormattedMessage id="detailsPanel.shardDetails.docLink" />
+                                            </ExternalLink>
+                                        </Box>
                                     </Box>
-                                </AccordionDetails>
-                            </Accordion>
-                        </Alert>
-                    </Grid>
-                )}
-
-                <Grid item xs={12}>
-                    <Box
-                        sx={{
-                            px: 1,
-                            py: 2,
-                            bgcolor: 'background.paper',
-                            borderRadius: '2px',
-                        }}
-                    >
-                        <span style={{ marginRight: 8, fontWeight: 500 }}>
-                            <FormattedMessage id="detailsPanel.shardDetails.id.label" />
-                        </span>
-
-                        <span>{shardDetails.id}</span>
-                    </Box>
-                </Grid>
-            </>
-        )
-    );
+                                </TableCell>
+                            </TableRow>
+                            <TableRow
+                                sx={{
+                                    background: theme.palette.background.paper,
+                                }}
+                            >
+                                {columns.map((column, index) => (
+                                    <TableCell key={`${column.field}-${index}`}>
+                                        <Typography>
+                                            {column.headerIntlKey && (
+                                                <FormattedMessage
+                                                    id={column.headerIntlKey}
+                                                />
+                                            )}
+                                        </Typography>
+                                    </TableCell>
+                                ))}
+                            </TableRow>
+                        </TableHead>
+
+                        <TableBody>
+                            {taskShards.map((shard) => (
+                                <TableRow
+                                    key={shard.spec.id}
+                                    sx={{
+                                        background: '#252526', // This is the hex code for the monaco editor background in dark mode.
```

Comment:
Given the details panel largely consists of Monaco editors, it made sense to have the shard information table follow suit in terms of background color.

That said, I am curious what capacity there is to introduce a color deeper than the _slate_ or _faux black_ of Estuary's color palette. The dark mode background colors used in the editor do not suit the application's color palette. _Thinking out loud._